generated/niscope/session.py

```diff
@@ -1448,28 +1448,28 @@ class _SessionBase(object):
     Specifies how the digitizer couples the trigger source. This property affects instrument operation only when  trigger_type is set to TriggerType.EDGE, TriggerType.HYSTERESIS, or TriggerType.WINDOW.
     '''
     trigger_delay_time = _attributes.AttributeViReal64TimeDeltaSeconds(1250015)
-    '''Type: datetime.timedelta
+    '''Type: float
 
     Specifies the trigger delay time in seconds. The trigger delay time is the length of time the digitizer waits  after it receives the trigger. The event that occurs when the trigger delay elapses is the Reference Event.
     Valid Values: 0.0 - 171.8
     '''
     trigger_from_pfi_delay = _attributes.AttributeViReal64TimeDeltaSeconds(1150052)
-    '''Type: datetime.timedelta
+    '''Type: float
```

Comment:
A lot of these got changed. If it means a lot of duplicate metadata, then we should chat about an alternative on what this is generated from.